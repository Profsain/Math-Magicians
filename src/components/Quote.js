import React from 'react';
import '../styles/Quote.css';
import HeaderNav from './Header';

export const Quote = () => {
  return (
    <div>
      <HeaderNav />
      <div className="Quote-main">
        <div className="Quote-left">
          <h2>Now is quote time!</h2>
          <img src="magic-quote.jpg" alt="Quote magic boy"/>
        </div>
        <div className="Quote-right">
          <p>
            <q>Without mathematics, there’s nothing you can do. Everything around you is mathematics. Everything around you is numbers.</q>
            <span><small>— Shakuntala Devi, Indian writer and mental calculator</small></span>
          </p>
        </div>
      </div>
    </div>
  );
}
