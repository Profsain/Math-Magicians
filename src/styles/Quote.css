.Quote-main {
  display: flex;
  flex-direction: row;
  justify-content: center;
  align-items: center;
}

.Quote-left img {
  width: 430px;
}

.Quote-left h2 {
  text-align: center;
  font-size: 2rem;
}

.Quote-right {
  width: 40%;
  font-size: 2rem;
}

span {
  font-size: 1.2rem;
}