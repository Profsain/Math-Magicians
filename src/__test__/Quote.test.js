import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

it('renders component correctly', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
