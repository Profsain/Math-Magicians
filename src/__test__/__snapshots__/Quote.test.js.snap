// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders component correctly 1`] = `
<div>
  <div
    className="Quote-main"
  >
    <div
      className="Quote-left"
    >
      <h2>
        Now is quote time! 
      </h2>
      <img
        alt="Quote magic boy"
        src="magic-quote.jpg"
      />
    </div>
    <div
      className="Quote-right"
    >
      <p>
        <q>
          Without mathematics, there’s nothing you can do. Everything around you is mathematics. Everything around you is numbers.
        </q>
        <span>
          <small>
            — Shakuntala Devi, Indian writer and mental calculator
          </small>
        </span>
      </p>
    </div>
  </div>
  <footer
    className="Footer"
  >
    <p>
      Developer: Husseini Mudi @Microverse Inc
    </p>
  </footer>
</div>
`;
